### DISCUSSION TITLE
Is Offline Tracking method accepted in MOT challange of CVPR2022？

### DISCUSSION THREAD

**Discussion by foolhard** [bdd100k/bdd100k#227] [Q&A] [answered]
I don't see any rule regarding Online Tracking or Offline Tracking in the CVPR2022 challenge webpage.

Do you accept Offline Tracking method for MOT challenge? 

https://www.bdd100k.com/challenges/cvpr2022/

**Comment by thomasehuang** [comment, marked as answer]
We will accept Offline Tracking methods for the challenge, but please indicate so in the name of the method during submission (add `-OF` at the end). Depending on the submissions, we may have separate tracks for online and offline trackers.
